extensions/amp-consent/0.1/test/test-consent-ui.js

```diff
@@ -49,10 +49,16 @@ describes.realWin('consent-ui', {
       getAmpDoc: () => {return ampdoc;},
       element: parent,
       win,
-      getViewport: () => {return {
-        addToFixedLayer: () => {},
-        removeFromFixedLayer: () => {},
-      };
+      getViewport: () => {
```

Comment:
There's no test coverage for handling received messages. Mostly because I didn't put one last time : ) Let's create a seperate PR to add them 😄